lib/chef/audit/audit_reporter.rb

```diff
@@ -59,6 +60,7 @@ def audit_phase_complete
       # known control groups.
       def audit_phase_failed(error)
         # The stacktrace information has already been logged elsewhere
+        @exception = error
```

Comment:
Having `exception` and `error` below is a little confusing.  I think renaming `@exception` to `@audit_phase_failed_error` makes it more clear what we stored.